eZ/Publish/Core/FieldType/RichText/Resources/schemas/docbook/ezpublish.rng

```diff
@@ -808,6 +757,18 @@
     </attribute>
   </define>
 
+  <define name="ez.custom.eztemplatetype">
```

Comment:
Open question is if we can also get rid of this attribute. As it means the tag isn't really plain custom tag as supported by migration from legacy, and by other parts of the system, it's still essentially a separate style tag as indicated by this attribute.

I don't know exactly how it could be done, but one way would be to align style config to custom tags, and the generic output we do for custom tags in html5-edit to accommodate for use in Styles _(for cases where own tag button is not needed)_.

But given the time we have now I guess this is ok, and we can much easier get rid of an attribute later if we want, compared to own style tag.

So 👍from my side